x/evm/keeper/precompiles.go

```diff
@@ -87,6 +88,15 @@ func AvailablePrecompiles(
 		panic(fmt.Errorf("failed to instantiate stride outpost: %w", err))
 	}
 
+	osmosisOutpost, err := osmosisoutpost.NewPrecompile(
+		"portID", "channelID",
```

Comment:
what are the correct port and channel IDs here @Vvaradinov?

Reply:
Channel: "channel-215"
Port: "transfer"

Here for ref:
https://testnet.evmos.explorers.guru/transaction/CDA0524CFD23C01359600EB53C4D1D6D214E86FD110663DC41669CDAB98351E1